package.json

```
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/iden3/circom_runtime#readme",
   "dependencies": {
-    "ffjavascript": "0.2.10",
+    "ffjavascript": "0.2.33",
```

Can this be changed to use semver, so this issue doesn't come up again when something changes in ffjavascript?
```suggestion
    "ffjavascript": "^0.2.33",
```